pipeline/runtime/run_test.go

```diff
@@ -0,0 +1,49 @@
+package runtime
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_splitCommands(t *testing.T) {
+	tests := []struct {
+		command string
+		want    []string
+	}{
+		{
```

Comment:
Can you test it with if/else clause, case, function etc. Basically all the possibilities in bash or /bin/sh. Otherwise, it may cause regression

Reply:
closing this PR. Parsing these possibilities just isn't possible without a full blown bash parser. The change should happen in the platform at the root of the problem, not in the lite-engine as patch work.